### ISSUE TITLE
Modernizr >= 2 and >= 3 use touchevents insted touch

### ISSUE THREAD

**Issue by evilebottnawi** [cloudfour/hideShowPassword#42]
Line 81:
```javascript
touchSupport: (typeof Modernizr === 'undefined') ? false : Modernizr.touch,
```

**Comment by tylersticka** [comment]
Good call, @evilebottnawi !

**Comment by tylersticka** [comment]
- [ ] Include updated Modernizr for demo
- [ ] Update Modernizr test
- [ ] Bump version in package info
- [ ] Update README
- [ ] Tag and push new version

**Comment by tylersticka** [comment]
@evilebottnawi Should be fixed now, version is `2.0.10`. Thanks again! :+1:
